oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -14,34 +14,33 @@
 // limitations under the License.
 //
 
-use std::{
-    fmt::{self, Display, Formatter},
-    string::String,
-};
-
-use log::{error, info, log};
-
-use oak_abi::OakStatus;
-use std::thread::{self, JoinHandle};
-
 use crate::{
     pretty_name_for_thread,
     runtime::{Handle, RuntimeProxy},
 };
-use oak_abi::proto::oak::log::{Level, LogMessage};
+use log::{error, info, log};
+use oak_abi::{
+    proto::oak::log::{Level, LogMessage},
+    OakStatus,
+};
 use prost::Message;
+use std::{
+    fmt::{self, Display, Formatter},
+    string::String,
+    thread::{self, JoinHandle},
+};
 
 pub struct LogNode {
-    config_name: String,
+    name: String,
```

Comment:
Should this field be called `display_name` (just like in `GrpcServerNode` and `WasmNode`) ?

Reply:
Done.